online-inference/fastertransformer/client/example.py

```diff
@@ -13,10 +13,16 @@
 
```

Comment:
Should encoding/decoding/tokenizing be the clients responsibility? Everything I've worked with so far that is handled server side, so all the clients need to do is send a curl request w/ some JSON which is simpler to demonstrate for examples.

Reply:
The goal was to isolate building the fastertransformer backend for Triton and not make it batteries included. The client examples are just for reference so that they can build out the client as per their requirements to go to production. Also there are two different models that we are encoding/decoding for so I thought it would be easier to see that hide away the implementation and leave them in the dark when they call a particular model on FT.